test/functionalTest/cases/3068_ngsi_v2_based_forwarding/legacy_forwarding_true_false_default_with_accumulator.test

```diff
@@ -0,0 +1,133 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Legacy Forwarding default value
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 38,186-187,235 IPV4
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Registration of E1 without Legacy Forwarding, for the accumulator - forward to be in NGSIv2 format
+# 02. Query of entities with id E1 - get empty response as the accumulator doesn't respond, but a forward is provoked
+# 03. Dump and reset the accumulator, see the forwarded query in NGSIv2 format
+#
+# FIXME: Due to a bug in Orion, the forward provoked by the query of step 2 makes the broker send not one but TWO forwarded
+#        request of the query to the accumulator.
+#        This is a bug of the broker that is also present in V1 forwarding and it is 'more or less acceptable', but should be fixed some day ...
+#
+#
+# FIXME: It seems like older versions of libcurl (CentOS 7 used version 7.29 of libcurl) contain a bug that makes the Content-Type
+#        always be added to the sent message, also when the Content-Type is EMPTY (which should inhibit this behaviour).
+#        The result of this bug is that the line
+#          Content-Type:
+#        is a part of forwarded messages.
+#        In newer versions of libcurl this haqs been fixed (Ubuntu 18.04 uses libcurl 7.58)
+#        For now, a "grep -v 'Content-Type'" is added to calls of accumulatorDump as a work-around.
+#        The real fix would be to use a newer version of libcurl in CentOS 7 (Travis, Jenkins)
```

Comment:
To use grep -v to hide this may make jenkins and travis happy, but the problem is still there and could potencially broke with real CPrs. Not a real fix of the problem.

Thus, I propose the following:

1. Review the changelog between 7.29 and 7.58 to confirm the bug and the version in which it has been solved. Let's name it version X. Some additional comments:
    * In my system the problem also happens. I'm using Debian with 7.38.0-4+deb8u9. Thus, I'd suggest to explore the delta from 7.38 to 7.58 (shorter than the delta from 7.29)
    * If the fix is not evident after exploring the changelog, then ask in the libcurl devel list or in SOF
2. Do a quick test in local with version X in a system wtich previously failed (mine could be a valid place)
3. Create a fresh branch from master and implement the fix there. The CB code will not be touched but fixes in documentation, dockerfile and travis will be needed.
4. Merge that branch in master, ensuring that CI e2e regression keep working
5. Upgrade this PR branch master (removing the grep -v hack)

While that comes, let's keep this PR on hold.

Reply:
The `grep -v` ensures that the only error in the test is the unwanted occurrence of "Content-Type:", everything else is as it should. That was the reason for the modification.

Question raised in SOF:
https://stackoverflow.com/questions/55588447/empty-content-type-in-curl-easy-perform-get-request

Reply:
So at the end, after research, this wasn't a problem with libcurl but something related with Flask and the libraries in which it depends.

Thus, NTC (happily :)